tests/test_resources.py

```diff
@@ -932,6 +932,9 @@ def test_subscribe_multiple_errors(self):
             sub_errs = err.errors['subscription.account.account_code']
             self.assertEqual(len(sub_errs), 2)
             self.assertEqual(type(sub_errs[1]), recurly.errors.ValidationError.Suberror)
+            # make sure we aren't falling back to default Suberror string
+            # represenation
+            self.assertFalse("Suberror" in str(err))
```

Comment:
Positive asserts (checking the presence of a string) or much more robust than non-presence checks, maybe add one or two ?

Reply:
python2 and 3 arrange the keys in a different order. I'll add something to sort them first so they are consistent.